tests/other/testUninstallActivateDeactivate.php

```diff
@@ -38,4 +34,24 @@ public function testUninstall() {
 		}
 
 	}
+
+	public function test_deactivate() {
+
+		$res = activate_plugin( 'backupwordpress/backupwordpress.php' );
+		$this->assertFalse( is_wp_error( $res ) );
+		deactivate_plugins( 'backupwordpress/backupwordpress.php' );
+		$this->assertFalse( is_plugin_active( 'backupwordpress/backupwordpress.php' ) );
+		$this->assertGreaterThanOrEqual( 1, did_action( 'deactivate_backupwordpress/backupwordpress.php' ) );
```

Comment:
Ah my mistake. I was thinking of `has_action()` which returns the priority of the callback.